### PR TITLE
Class highlight

### PR DESCRIPTION
**Pull request by akjw**

### CODE REVIEW

File: syntaxes/bl4.tmLanguage.json [hunk 1]
```diff
@@ -5,14 +5,31 @@
...
+    },
+    {
+      "include": "#class-dec"
     }
   ],
   "repository": {
+    "keywords": {
+      "patterns": [
+        {
+          "name": "keyword.control.l4",
```
Comment (kharus) [inline syntaxes/bl4.tmLanguage.json]: `keyword.control.bl4`

File: syntaxes/bl4.tmLanguage.json [hunk 2]
```diff
@@ -39,10 +60,43 @@
...
+    "class-dec": {
+      "name": "meta.entity.class.l4",
+      "begin": "(class)\\s+(\\w+)",
+      "end": "(?=})",
+      "captures": {
+        "1": {
+          "name": "storage.type.class.l4"
+        },
+        "2": {
+          "name": "variable.name"
```
Comment (kharus) [inline syntaxes/bl4.tmLanguage.json]: consider `entity.name.type.bl4`

File: syntaxes/bl4.tmLanguage.json [hunk 3]
```diff
@@ -27,6 +44,10 @@
         }
       ]
     },
+    "type": {
+      "name": "entity.name.type",
+      "match": "(?<=\\:|->)\\s+?\\w+|\\((\\w+\\,\\s*\\w+)\\)"
```
Comment (kharus) [inline syntaxes/bl4.tmLanguage.json:50]: Can you please add a comment for this one :-) ?

Comment (akjw) [reply syntaxes/bl4.tmLanguage.json:50]: done!

**Pull request merged**